lib/services/extensions-service-base.ts

```diff
@@ -52,7 +52,7 @@ export class ExtensionsServiceBase {
 
 			if( this.shouldUpdatePackage(cachedVersion, extensionVersion) ) {
 				this.$logger.printInfoMessageOnSameLine(`Updating ${packageName} package...`);
-				let zipFileName = temp.path({ path:  path.join(this.cacheDir, packageName + ".zip") });
+				let zipFileName = path.join(this.cacheDir, packageName + ".zip");
```

Comment:
Also the `temp` module is not used anymore in this file, so the `import` statement in its beginning becomes obsolete